ArcGISRuntimeSDKQt_CppSamples/Layers/DisplayAnnotation/README.metadata.json

```diff
@@ -0,0 +1,29 @@
+{
+    "category": "Layers",
+    "description": "Display annotation from a feature service URL.",
+    "ignore": false,
+    "images": [
+        "screenshot.png"
+    ],
+    "keywords": [
+        "AnnotationLayer",
+		"FeatureLayer",
```

Comment:
there are still lots of tab vs space issues in both metadata.json files. Please use spaces only